features/filesystem/sd/SDBlockDevice.cpp

```diff
@@ -477,6 +509,57 @@ int SDBlockDevice::_cmd8() {
     return -1; // timeout
 }
 
+/* SDBlockDevice::_go_idle_state()
```

Comment:
All documentation in headers please in doxygen format. Private members can exclude the additional * for now that indicates doxygen should render